internal/runtime/client/client.go

```diff
@@ -153,6 +160,89 @@ func (c *client) Unregister(extensionConfig *runtimev1.ExtensionConfig) error {
 	return nil
 }
 
+func (c *client) CallAll(ctx context.Context, hook catalog.Hook, request runtime.Object, response runtimehooksv1.AggregatableResponse) error {
```

Comment:
Please add the godoc comments from the interface here and on Call as well.
I think we also need some documentation about under which circumstances we return an error and how we modify the response based on failurePolicy, ... (for both Call and CallAll)